packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -27,9 +27,9 @@ import '../runner/flutter_command.dart';
 /// ```
 const Map<String, String> _kManuallyPinnedDependencies = <String, String>{
   // Add pinned packages here.
-  'mockito': '3.0.0-beta', // TODO(aam): https://github.com/dart-lang/mockito/issues/110
-  'matcher': '0.12.2+1', // TODO(ianh): https://github.com/flutter/flutter/issues/18608, https://github.com/dart-lang/matcher/pull/88
-  'flutter_gallery_assets': '0.1.4', // See //examples/flutter_gallery/pubspec.yaml
+  'mockito': '3.0.0',
+  'matcher': '0.12.3+1', // TODO(ianh): https://github.com/flutter/flutter/issues/18608
```

Comment:
Please don't do this. It's a breaking change for our users.

Reply:
Guess this issue has been resolved already by https://github.com/flutter/flutter/pull/20602